src/Microsoft.PowerShell.Commands.Utility/commands/utility/ReadConsoleCmdlet.cs

```diff
@@ -14,7 +14,6 @@
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    ///
     /// Retrieves input from the host virtual console and writes it to the pipeline output.
     ///
```

Comment:
The regular expression should remove this whitespace unless the line after the whitespace contains text and not `<`.

Reply:
Fixed.